src/common/Browser.ts

```diff
@@ -293,9 +293,16 @@ export class Browser extends EventEmitter {
    * })();
    * ```
    */
-  async createIncognitoBrowserContext(): Promise<BrowserContext> {
+  async createIncognitoBrowserContext(
+    proxyServer?: string,
+    proxyBypassList?: string
```

Comment:
Ah sorry, one more thing: Could you change this to be an object instead of normal arguments. `createIncognitoBrowserContext` will probably accept other arguments in the future too so this might be a good way to prevent a breaking change later.